tests/framework/installer/ceph_installer.go

```diff
@@ -123,7 +123,7 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 			return fmt.Errorf("failed to create namespace %q. %v", namespace, err)
 		}
 	}
-	if !utils.IsPlatformOpenShift() {
+	if !utils.IsPlatformOpenShift() && namespace != "upgrade-ns-system" {
```

Comment:
How about adding a comment about why we disable it during the upgrade test. In a future release we should then be able to enable this again when we don't have the controller runtime compatibility issue in the versions we're testing.

Reply:
Done.